tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -349,6 +361,14 @@ def test_forward_convolution():
         _test_convolution('depthwise', [4, 124, 17, 17], [1, 1, 124, 1], [1, 1], [1, 1], 'SAME', 'NCHW')
         _test_convolution('depthwise', [4, 12, 17, 17], [3, 3, 12, 1], [1, 1], [2, 2], 'VALID', 'NCHW')
         _test_convolution('depthwise', [4, 12, 17, 17], [3, 3, 12, 2], [1, 1], [2, 2], 'VALID', 'NCHW')
+        _test_convolution('conv_transpose', [4, 32, 8, 8], [1, 1, 176, 32], [1, 1], [1, 1], 'SAME',
```

Comment:
please add test case for cpu

Reply:
NHWC layout is not supported, see [nn.conv2d_transpose](https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/op/nn/_nn.py#L263). Is it still needed to add test cases for cpu?

Reply:
yeah, test cases for cpu are needed. How aboud adding test cases for NCHW input, and adding layout transformation to get the expected NCHW layout for NHWC input?

Reply:
Do you mean adding the transformation from NHWC to NCHW if 'conv2d_transpose' op at the beginning of _conv ?

Reply:
yeah, if the layout is NCHW, the nn.conv2d_transpose should work well. If layout is NHWC, then we can consider transforming the layout to NCHW to be compatible with nn.conv2d_transpose

Reply:
I have added transformation and the cpu test cases, could you please review again? Thanks!